game/state/gamestate_serialize.xml

```diff
@@ -1,2064 +1,2065 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openapoc_gamestate>
-	<object external="true">
```

Comment:
I guess you used an auto-formatter on the xml file here? What tool did you use? It makes sense to have consistency, otherwise it'll cause pretty much every line to change any time someone edits anything, like here.

And even if reformatting the xml is a good idea, it should probably be a separate PR completely, as otherwise 'git blame' will point out this commit for every modified line, which is less than helpful.

Reply:
I used Visual studio and it formatted this file. I didn't mean to change the formatting only add one item to agent. I haven't worked much with xml so I will need to check if I can get the xmllint working with visual studio.

Reply:
It seems we are using tabs and indentation of 8 for xml? gamestate_serialize.xml seems fixed now.